ReadMe.md

```diff
@@ -0,0 +1,86 @@
+## Quick Instructions
+First of all we need to Install MARS Environment:
+     
+We need to perform the following steps without usings sudo (To avoid generating files and folders owned by root)
+Now Open The Terminal or Command Prompt and follow the following steps step-by-step:
+
+1) Basic Initial Steps: 
+      
+    apt-get update
+    apt-get install -y build-essential ruby ruby-dev sudo wget
+          
+2) Download the autoproj bootstrap script:
+      
+    wget http://rock-robotics.org/autoproj_bootstrap
+          
+    
+3) Run the boostrap script and provide the repository address:
+
+The user has to set up his/her keys in the github and gitlab account and use the correspondent command:
+      
+    ruby autoproj_bootstrap git git@github.com:jura02/simulation-buildconf.git branch=envireMars          
+ 
+If the user has not generated the SSH key in his/her github and gitlab account then use the correspondent command: 
+    
+    ruby autoproj_bootstrap git https://github.com/jura02/simulation-buildconf.git branch=envireMars
+        
+
+Answer "whether C++11 should be enabled for Rock packages [false]" with "true" and the other with default (just hit enter)
+    
+Sometimes you may face the problems like "No Access Rights to your repository": Make sure you have corrected generated the SSH Key for both GitLab and GitHub
```

Comment:
Please Move all this to the Troubleshooting section:

> Sometimes you may face the problems like "No Access Rights to your repository": Make sure you have corrected generated the SSH Key for both GitLab and GitHub

> Similarly, sometimes you don't have access to the specific Repository forexample:
"No Access Rights to git@git.hb.dfki.de/models-environments" 
In this case, you have to make an access request to the maintainer/owner of that Repository.

Reply:
Done.